src/azul/service/manifest_service.py

```diff
@@ -623,6 +628,7 @@ def write_to(self, output: IO[str]) -> Optional[str]:
                 for doc_path, column_mapping in self.manifest_config.items():
                     entities = self._get_entities(doc_path, doc)
                     self._extract_fields(entities, column_mapping, row)
+                row['drs_url'] = self._drs_url(one(doc['contents']['files']))
```

Comment:
Is it possible for the doc[`content`][`files`] to have more than one value? Should we attempt to handle an error if it can?

Reply:
My understanding is that it will only ever have one value in the `files` index but can have multiple values when iterating over the other indices. This usage of `more_itertools.one` is mirrored on multiple similar usages throughout the `manifest_service` module, so if there's a better way to do it, no one else as come up with it yet.